### ISSUE TITLE
Issue with responsive layout

### ISSUE THREAD

**Issue by IngDante** [malihu/malihu-custom-scrollbar-plugin#479]
Hello there!

I have just discovered this plugin and...thanks for this malihu, I love it!

I am playing with ["Responsive custom scrollbar with CSS3 media queries"](http://manos.malihu.gr/code-example/responsive-custom-scrollbar-with-css3-media-queries/),
more or less just using the [sample page](http://manos.malihu.gr/repository/custom-scrollbar/code-examples/responsive-custom-scrollbar-with-css3-media-queries.html).
(for anyone reading this that may want to fiddle with it...[here is a fiddle](http://jsfiddle.net/jcnszzka/)!)

The question I have is: why does the last `<li>` element go to a new "line"?

I am fairly sure it has to do with the style (the width/height of the container maybe?), and it's possibly a very easy thing to "fix", but I played around with it with no luck.

I then decided to try the Javascript way: I set `axis:"yx"` to `axis:"x"` and that indeed forces the `<li>`s to stay on one line.

The drawback of course is that when you resize the window you don't get the vertical scrollbar.
That looked like something that could be handled by using the `update` method: I defined the callback `onUpdate` and made it set the axis to `y` to re-render the whole thing, but I guess I am doing something wrong because...nothing happens! (to be honest, it looks like the callback doesn't even get called)

So, to recap:
* I would like the scrollbar to have the `xy` responsive behaviour, but staying in 1 line
* I would like to achieve that using just CSS (although I suspect going the Javascript way could be an option, despite my lack of luck with `update`)

Am I missing something or there is a bug of some sort?

Thanks!
